lms/templates/seq_module.html

```diff
@@ -3,6 +3,16 @@
 
 <div id="sequence_${element_id}" class="sequence" data-id="${item_id}" data-position="${position}" data-ajax-url="${ajax_url}" data-next-url="${next_url}" data-prev-url="${prev_url}">
   <div class="path"></div>
+    % if masquerade_override:
```

Comment:
Went with `overriding_hidden_exam`, as it explains the displayed message. LMK if you have a better suggestion.